subprojects/plugin-use/src/integTest/groovy/org/gradle/plugin/use/AlreadyOnClasspathPluginUseIntegrationTest.groovy

```diff
@@ -395,7 +395,7 @@ class AlreadyOnClasspathPluginUseIntegrationTest extends AbstractIntegrationSpec
             publishing { repositories { maven { url = uri("../$localPluginRepoPath") } } }
         """.stripIndent()
 
-        succeeds "-b", "$pluginBundleName/build.gradle", "publish"
+        executer.inDirectory(file(pluginBundleName)).withTasks("publish").run()
```

Comment:
Attempt to use qualified task names instead of running a task within a subdirectory.

Reply:
In this case I am pointing Gradle to the root of a project to be built, not to a subdirectory to run a task from.